tests/hpc/slurm_master_backup_db.pm

```diff
@@ -0,0 +1,69 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: HPC_Module: slurm failover - database
+#    This test is setting up slurm control backup nodes with accounting
+#    configured (database) so that the failover of the ctls can be tested
+# Maintainer: Sebastian Chlad <sebastian.chlad@suse.com>
+
+use base "hpcbase";
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use utils;
+
+sub run {
+    my $self  = shift;
+    my $nodes = get_required_var("CLUSTER_NODES");
+    $self->prepare_user_and_group();
+
+    zypper_call('in nfs-client rpcbind');
+    systemctl 'start nfs';
+    systemctl 'start rpcbind';
+    assert_script_run("showmount -e 10.0.2.1");
+    assert_script_run("mkdir -p /shared/slurm");
+    assert_script_run("chown -Rcv slurm:slurm /shared/slurm");
+    assert_script_run("mount -t nfs -o nfsvers=3 10.0.2.1:/nfs/shared /shared/slurm");
+
+    # provision HPC cluster, so the proper rpms are installed
+    # and proper services are enabled and started
+    zypper_call('in slurm slurm-munge slurm-slurmdbd');
+    barrier_wait("SLURM_SETUP_DONE");
+    barrier_wait("SLURM_MASTER_SERVICE_ENABLED");
+    assert_script_run("cat /etc/slurm/slurm.conf");
```

Comment:
this doesn't do anything than printing the output to the serial terminal log, which is ok. But it might be also useful to use record_info with script output for this kind of calls. E.g.

    record_info('slurm conf', script_output('cat /etc/slurm/slurm.conf'));

Reply:
sure. I add this silly cat just to make sure the config is as it should be.
But overall I like your suggestion overall. It is good to have config in case we report a bug.

Reply:
@jlausuch I do follow your suggestion. I like it actually.